scripts/install-using-catalog-source.sh

```diff
@@ -29,5 +30,5 @@ if ! [[ "${subscription_types[@]}" =~ (^|[[:space:]])"$subscription_type"($|[[:s
 fi
 
 sed -e "s,quay.io/openshift/federation-operator-registry,quay.io/$registry/federation-operator-registry," $dir/olm-testing/catalog-source.yaml | oc apply -f -
```

Comment:
```suggestion
sed -e "s,quay.io/openshift/federation-operator-registry,quay.io/$registry/federation-operator-registry," $dir/olm-testing/catalog-source.yaml | $kubectl apply -f -
```

I also forgot this one the first time I edited the file LOL